### PR TITLE
Add ACK mode for VC

### PR DESCRIPTION
**Pull request by charleszheng44**
1. allow Virtualcluster to be exposed through loadbalancer.
2. when vc is exposed by loadbalancer, the vcctl will replace vc url
in kubeconfig by external ip of the loadbalancer.
3. add ack specified yaml files that pulling container images from docker registry
hosts alibaba cloud. (ack users may have issues with downloading image
from docker hub).

### CODE REVIEW

File: incubator/virtualcluster/cmd/vcctl/subcmd/create.go [hunk 1]
```diff
@@ -218,16 +219,28 @@ func genKubeConfig(clusterNamespace, vcKbCfg string, cli client.Client, minikube
...
-func replaceServerAddr(kubeCfgContent []byte, cli client.Client, clusterNamespace string) ([]byte, error) {
-	minikubeIP, err := getMinikubeIP()
-	if err != nil {
-		return nil, err
-	}
-	svcNodePort, err := netutil.GetSvcNodePort(APIServerSvcName, clusterNamespace, cli)
-	if err != nil {
-		return nil, err
+func replaceServerAddr(kubeCfgContent []byte, cli client.Client, clusterNamespace, mode string) ([]byte, error) {
+	var newStr string
```
Comment (zhuangqh) [inline incubator/virtualcluster/cmd/vcctl/subcmd/create.go]: what if mode == "" will vc-manager do

Comment (charleszheng44) [reply incubator/virtualcluster/cmd/vcctl/subcmd/create.go]: we decide to replace `minikube` and `ack` mode by more general `nodeport` and `loadbalancer` mode.

**Pull request merged**